package.json

```diff
@@ -17,7 +17,7 @@
     "test:single": "mocha --require withDom.js --compilers js:babel-core/register --watch --reporter dot",
     "test:watch": "mocha --require withDom.js --compilers js:babel-core/register --recursive test/*.js --watch --reporter dot",
     "test:env": "sh ./example-test.sh",
-    "test:all": "npm run react:13 && npm test && npm run react:14 && npm test && npm run react:15 && npm test",
+    "test:all": "npm run lint && npm run react:13 && npm run test:only && npm run react:14 && npm run test:only && npm run react:15 && npm run test:only",
```

Comment:
Just to clarify, we'd have something like:
```
"pretest": "npm run lint",
"test": "npm run test:only",
"check": "npm run test:all", 
"test:all": "npm run pretest && npm run react:13 && npm run test:only && npm run react:14 && npm run test:only && npm run react:15 && npm run test:only"
```

I'm not clear on what purpose `check` serves. Before my changes, it was just running lint a fourth time on top of the three times `test:all` ran lint via 3 calls to `test`.